envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -121,7 +121,11 @@ private fun Value?.toIncomingTimeoutPolicy(): Incoming.TimeoutPolicy {
         ?.takeIf { it.isNotBlank() }
         ?.let { Durations.parse(it) }
 
-    return Incoming.TimeoutPolicy(idleTimeout, responseTimeout)
+    val connectionIdleTimeout: Duration? = this?.field("connectionIdleTimeout")?.stringValue
+        ?.takeIf { it.isNotBlank() }
+        ?.let { Durations.parse(it) }
```

Comment:
Can `parse` throw? How do we handle errors in other places?

Reply:
Yes, parse can throw ParseException, but as I see for example in some lines above we do not handle it

Reply:
I'll add an issue for that.

Reply:
@slonka don't do it, we handle it in NodeMetadataValidator

Reply:
@slonka on the other hand, maybe we should wrap it in NodeMetadataValidationException